leabra7/events.py

```diff
@@ -173,11 +192,67 @@ def from_name(cls, freq_name: str) -> "Frequency":
 
 
 CycleFreq = Frequency(name="cycle", end_event_type=Cycle)
-TrialFreq = Frequency(name="trial", end_event_type=EndPlusPhase)
+TrialFreq = Frequency(name="trial", end_event_type=EndTrial)
 EpochFreq = Frequency(name="epoch", end_event_type=EndEpoch)
 BatchFreq = Frequency(name="batch", end_event_type=EndBatch)
 
 
+class PhaseType(Enum):
+    plus = 1
```

Comment:
In nearly every programming style, enums are in all caps (so PLUS, MINUS, ONE) because they are constants.
Also, I think you can use a unique values feature of the enum to communicate that the exact values don't matter.